src/relion/cryolo_relion_it/icebreaker_histogram.py

```diff
@@ -0,0 +1,52 @@
+import gemmi
+import json
+import matplotlib.pyplot as plt
+from pathlib import Path
+import plotly.express as px
+import pandas as pd
+
+
+def create_json_histogram(working_directory):
+    icebreaker_path = str(working_directory / "External/Icebreaker_group/")
+    json_hist_file_name = "ice_hist.json"
+    if Path(icebreaker_path + "/particles.star").is_file():
+        data = extract_ice_column(icebreaker_path + "/particles.star")
+    else:
+        return None
+    if data is None:
+        print("No data extracted.")
```

Comment:
Check warning suggests the case `data is None` could be covered by a test. Should we log a warning message here? Do we still want to go through the rest of the function?
Similar: what should we do if the file doesn't exist? Is a silent return appropriate?